### PR TITLE
Add mayhem support

### PR DESCRIPTION
**Pull request by Cl4sm**
- Add test_seed
- Add mayhem support to upx

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+project: wgibbs15/upx
```
Comment (xansec) [inline Mayhemfile]: Please omit the owner name for the project, as this will change after merging.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# Target name (should be unique within the project)
+target: upx
+
+# Base image to run the binary in.
+image: ghcr.io/wgibbs15/upx-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: You can omit the image name as well.

**Review by xansec — CHANGES_REQUESTED** [review]
Please do not commit binary files. If you are trying to seed your corpus, you can do this by specifying a tests/testsuite location in the Mayhemfile, or copying your seed file to the `/testsuite` directory in your Dockerfile.

**Comment by Cl4sm** [comment]
Removed the binary file and added cat to /testsuite folder in the dockerfile

**Comment by xansec** [comment]
Great, LGTM!

**Pull request merged**